src/sdk/pynni/nni/compression/torch/weight_rank_filter_pruners.py

```diff
@@ -178,7 +184,7 @@ class FPGMPruner(WeightRankFilterPruner):
     https://arxiv.org/pdf/1811.00250.pdf
     """
 
-    def __init__(self, model, config_list, optimizer):
+    def __init__(self, model, config_list, optimizer=None):
```

Comment:
FPGM is also an iterative pruner, optimizer should not be None

Reply:
Thanks, fixed.